mappings/net/minecraft/nbt/AbstractNbtNumber.mapping

```diff
@@ -1,7 +1,7 @@
 CLASS net/minecraft/class_2514 net/minecraft/nbt/AbstractNbtNumber
 	COMMENT Represents an NBT number.
 	COMMENT <p>
-	COMMENT The type {@linkplain net.fabricmc.yarn.constants.NbtTypeIds#NUMBER NbtTypeIds.NUMBER} can be used to
+	COMMENT The type {@link NbtElement#NUMBER_TYPE NUMBER} can be used to
```

Comment:
```suggestion
	COMMENT The type {@link NbtElement#NUMBER_TYPE NUMBER_TYPE} can be used to
```
since this is not plain, prefer to stay close to the actual field name